auctions/models.py

```diff
@@ -136,24 +138,17 @@ def is_biddable_by(self, user):
 
 @receiver(post_save, sender=Bid)
 def notify_matching_askers(sender, instance, **kwargs):
-    # TODO: make a nicer HTML email template
-    ASKER_NOTIFICATION_EMAIL_STRING = """
-    Bidders have met your asking price for {url}.
-    If you fix the issue, you may claim the payout by visiting the issue url:
-    {url}
-    """
-
+    email_template = get_template('../templates/email/ask_met.html')
     unnotified_asks = Bid.objects.filter(
         url=instance.url, ask_match_sent=None).exclude(ask__lte=0)
 
     for bid in unnotified_asks:
         if bid.ask_met():
+            email_context = {'ask': bid.ask, 'url': bid.url}
+            message = email_template.render(email_context)
             send_mail(
-                "[codesy] Your ask for %(ask)d for %(url)s has been met" %
-                (
-                    {'ask': bid.ask, 'url': bid.url}
-                ),
-                ASKER_NOTIFICATION_EMAIL_STRING.format(url=bid.url),
+                "[codesy] Your ask has been met!",
```

Comment:
I think we should keep the ask amount in the subject line. It's a good call-to-action to actually open the email. I.e., "We have $50 for you!"